test/test_functional_tensor.py

```diff
@@ -20,6 +20,8 @@ def _create_data(self, height=3, width=3, channels=3):
 
     def compareTensorToPIL(self, tensor, pil_image, msg=None):
         pil_tensor = torch.as_tensor(np.array(pil_image).transpose((2, 0, 1)))
+        if pil_tensor.dtype != tensor.dtype:
+            pil_tensor = pil_tensor.to(tensor.dtype)
```

Comment:
Which case led you to have to perform a casting here, is it for the extra `float32` and `float64` types?

I think it might be preferable to avoid doing implicit casts in this function, and instead perform the cast directly in the caller -- it's more explicit.

Reply:
Right, I can cast explicitly before instead of this function 👍